### ISSUE TITLE
reproduce scores from paper

### ISSUE THREAD

**Issue by tilmanbeck** [JayKumarr/OSDM#1]
Hi,

I tried to reproduce the scores reported in the paper using the Tweets in the data folder (I guess it is the Ts dataset referred to in the paper?) and the same parameters as reported.

Unfortunately, I get very low scores for all the metrics and was wondering what might be the problem. 
Did you use the same parameters for all datasets?
Is the code the same as you used for your final experiments reported in the paper?

Best,
Tilman

**Comment by JayKumarr** [comment]
Hi, 
Thank you for taking interest to reproduce the results. I have uploaded the results excel file in this folder. I set same parameter for all the datasets. But i just noticed that the actual power value on beta is "-4", but in the paper it is "-5".

**Comment by tilmanbeck** [comment]
Thanks for your quick response! 
I have also noticed the beta-parameter difference and fixed that already.

However, I am interested in how you perform the experiment and the evaluation. I did run the code without modification on the Tweets dataset and get 152 clusters (with one very big and multiple single clusters). For evaluation I compare the output file (format: docId -> clusterId) with the clusterNo in the Tweets data file.
Did I miss anything?
